packages/console/src/containers/ConsoleRoutes/internal.ts

```diff
@@ -0,0 +1,10 @@
+import safeLazy from '@/utils/lazy';
+
+export const __Internal__ImportError = safeLazy(async () => {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  const module = await import(
+    /* @vite-ignore */ `${window.location.origin}/some-non-existing-path`
+  );
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+  return module;
+});
```

Comment:
It would be better if we can add some comments to explain the purpose of this component

Reply:
cool will do

Reply:
Not blocking, tho